mozci/task.py

```diff
@@ -220,6 +220,10 @@ def is_backfill(self) -> bool:
     def is_retrigger(self) -> bool:
         return self.tags.get("action", "").startswith("retrigger-")
 
+    @property
+    def is_tests_grouped(self) -> bool:
```

Comment:
Nit: could you use this function in the `label_summaries` function in push.py too?